src/status_im/contexts/centralized_metrics/tracking.cljs

```diff
@@ -43,15 +43,31 @@
     :screen/onboarding.syncing-progress
     :screen/onboarding.syncing-progress-intro
     :screen/onboarding.syncing-results
-    :screen/onboarding.welcome})
+    :screen/onboarding.welcome
+
+    ;; Collectibles
+    :screen/wallet.collectible})
 
 (defn track-view-id-event
   [view-id]
   (when (contains? view-ids-to-track view-id)
     (navigation-event (name view-id))))
 
+(defn collectilbes-fetched-event
+  [db]
+  (let [accounts               (get-in db [:wallet :accounts])
+        amount-on-all-accounts (reduce (fn [collectibles-amount account]
+                                         (+ collectibles-amount (:current-collectible-idx account)))
```

Comment:
Yes, it is intended. It doesn't reflect the total amount but reflects our current knowledge about account and I think that should be ok. As far as I know our api doesn't provide total amount of collectibles before we fetched them.

Reply:
I guess in this case the value of sending the "total amount" metric will be lost, since we won't know whether the users have x amount of collectibles, we'll just get the first fetch amount unless they scroll till the end. Is it even worth sending it? What will we learn from this metric?